hazelcast/src/main/java/com/hazelcast/config/alto/AltoConfig.java

```diff
@@ -5,11 +5,8 @@
 import java.util.Objects;
 
 public class AltoConfig {
-    private static final boolean DEFAULT_ENABLED = false;
-    private static final int DEFAULT_EVENTLOOP_COUNT = Runtime.getRuntime().availableProcessors();
-
-    private boolean enabled = DEFAULT_ENABLED;
-    private int eventloopCount = DEFAULT_EVENTLOOP_COUNT;
+    private boolean enabled = false;
+    private int eventloopCount = Runtime.getRuntime().availableProcessors();
```

Comment:
Whatever kind of mechanism is used, the default should be controllable through system properties.

Reply:
https://github.com/hazelcast/hazelcast/pull/23319#discussion_r1064336149

Reply:
From the code it is clear that no System property is used to configure the eventloopCount.

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/009609eec0df96ba2a9327217aa40ce9d50c417b